src/components/IsLoggedIn/IsLoggedIn.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { isLoggedIn } from "app/selectors";
+
+const IsLoggedIn = ({ children }) => {
+  const userIsLoggedIn = useSelector(isLoggedIn);
+  const visitURL = useSelector(state => {
+    if (!userIsLoggedIn) {
+      const root = state.root;
+      if (root && root.visitURL) {
+        return root.visitURL;
+      }
+    }
+  });
+
+  if (!userIsLoggedIn) {
+    return (
+      <div>
+        Please{" "}
+        <a href={visitURL} rel="noopener noreferrer" target="_blank">
```

Comment:
This should be a vanilla link component.

Reply:
This is only a temporary page to make qa'ing and developing easier. There is a design for this page that's not this :D